xds/internal/server/listener_wrapper.go

```diff
@@ -217,7 +317,6 @@ func (l *listenerWrapper) Accept() (net.Conn, error) {
 			// us to stop serving.
 			return nil, fmt.Errorf("received connection with non-TCP address (local: %T, remote %T)", conn.LocalAddr(), conn.RemoteAddr())
 		}
-
 		l.mu.RLock()
```

Comment:
Please add the newline above the lock. It helps with quickly figuring out where the lock is grabbed when skimming through the code.

Reply:
Done.